ow_lander/scripts/path_planning_commander.py

```diff
@@ -25,59 +25,81 @@
 import activity_deliver_sample
 import activity_grind
 
-# === MAIN COMMANDER CLASS =============================
-class MoveGroupPythonInteface(object):
+class PathPlanningCommander(object):
   
   def __init__(self):
-    super(MoveGroupPythonInteface, self).__init__()
+    super(PathPlanningCommander, self).__init__()
     moveit_commander.roscpp_initialize(sys.argv)
     robot = moveit_commander.RobotCommander()
     scene = moveit_commander.PlanningSceneInterface()
 
     move_arm = moveit_commander.MoveGroupCommander("arm")
-    move_limbs = moveit_commander.MoveGroupCommander("limbs")
-    move_grinder = moveit_commander.MoveGroupCommander("grinder")
```

Comment:
Why do you eliminate these two groups?

Reply:
These would need to be done as part of OCEANWATER-536 and OCEANWATER-537 after ticket OCEANWATER-530 has been implemented